common/src/main/java/gov/cms/ab2d/common/model/Consent.java

```diff
@@ -19,7 +19,7 @@
 
     private String hicn;
 
-    private OffsetDateTime effectiveDate;
+    private LocalDate effectiveDate;
```

Comment:
Changed effectiveDate to LocalDate as the sample input (and the reference implementation) had only 8 characters for date - yyyyMMdd. (No timezone or timestamp etc...)